aries-site/src/pages/foundation/color.mdx

```diff
@@ -1,13 +1,117 @@
+
+For more direction on how to use background colors, see [background colors guidance](/foundation/background-colors-guidance).
+
+<BackgroundSwatch />
+
+#### Overlay
+
+The overlay color dims inactive content to direct attention to an active layer. Apply it behind center layer, side drawer layer, or dialog surfaces to reduce cognitive load using `color.background.screenOverlay`.
+
+### Borders
+Border colors delineate areas only when spacing or background contrast are insufficient. Use `color.border.default` for neutral boundaries and dividers, and omit borders where existing spacing or contrast already provides clear separation.
+
+<BorderSwatch />
+
+### Text
+Text colors help create hierarchy and ensure readability. Use `color.text.default` for body copy and `color.text.heading` for headings. `color.text.strong` can be used to emphasize key figures or other important text. To de-emphasize content and create a clear visual hierarchy, apply `color.text.weak` to secondary information.
+
+<TextSwatch />
+
+### Icon
+Icon colors follow text hierarchy and must remain legible at small sizes. Match an icon's color emphasis with its accompanying text.
+
+<IconSwatch />
+
+### Decorative
+Decorative colors provide brand and accent without conveying status or interaction. Limit secondary decorative hues to illustration and non‑interactive ornamentation. Avoid substituting decorative colors for status; use semantic text or foreground status design tokens instead.
+
+<DecorativeSwatch />
+
+### Data visualization
+Data visualization colors differentiate categories without overwhelming perception. This palette has been crafted to ensure each series is visually distinct and color blind friendly. Apply data visualization colors according to the following guidelines:
+
+- Begin categorical series with the first design token and proceed sequentially.
+- Consider consolidating series with limited data into an "other" category represented by a single data visualization color; provide the ability to drill-down to view detail.
+- When using HPE Design System DataChart, series colors are applied automatically in sequence to support consistent interpretation.
+
+<DataVisSwatch />
+
+### Foreground
+Foreground colors fill graphic and quantitative regions (meters, progress, indicators) with semantic meaning.
+
+<ForegroundSwatch />
+
+### Status colors
+Status colors reinforce state and feedback (critical, warning, ok, informational, unknown) across surfaces, text and icons.
+
+- Use status color only when conveying semantic meaning; avoid decorative substitution.
+- Pair status color with an icon or concise label for rapid scanning.
```

Comment:
Wondering if we mention that it should be paired with an icon/label to meet accessibility?